src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/Manifest.component.tsx

```diff
@@ -102,10 +102,11 @@ function ManifestComponent({
                     getDesiredManifestResource(appDetails, params.podName, params.nodeType),
             ])
                 .then((response) => {
-                    let _manifest
+                    let _manifest: string
                     if (
                         appDetails.appType === AppType.EXTERNAL_HELM_CHART ||
                         appDetails.deploymentAppType === DeploymentAppType.helm ||
+                        appDetails.deploymentAppType === DeploymentAppType.argo_cd ||
```

Comment:
After adding this condition will there be any else case? If no, can we remove the if else?

Reply:
removed the else part, did sanity for the same.